tests/test_core.py

```diff
@@ -170,6 +170,19 @@ def test_create_interpolator(simple_grid):
     assert np.allclose(interp(field), linear_value)
 
 
+
+def test_set_cell_volumes():
+    """
+    Test that cell volumes are set properly.
+    """
+    grid_conf2 = {"r_min": 0,
+                  "r_max": 0.1,
+                  "dr": 0.1/7,
+                  coordinate_system = 'cartesian'}
```

Comment:
According to Travis CI this is invalid syntax, use double ==

Reply:
No, the problem is not that you need double =, it's that the last item in the dictionary is not in the appropriate syntax for a dictionary item. No equal sign at all.
```suggestion
                  "coordinate_system": 'cartesian'}
```

Reply:
I fixed that right off the bat before I could even start with the test. The test was not just busywork.  It helped me find a bug in the spherical coordinate section of the code. :)